remove reference of renameModelFields (#7838)
aws-amplify/docs: src/pages/[platform]/build-a-backend/data/connect-to-existing-data-sources/connect-postgres-mysql-database/index.mdx

## Rename generated models and fields

> To improve the ergonomics of your API, you might want to rename the generate fields or types to better accommodate your use case. Use the `renameModels()` modifier to rename the auto-inferred data models.

Reviewer: ```suggestion
To improve the ergonomics of your API, you may want to rename the generated types to better accommodate your use case. Use the `renameModels()` modifier to rename the auto-inferred data models.
```
